unix.c

```diff
@@ -612,7 +612,7 @@ enet_socket_receive (ENetSocket socket,
 #ifdef NO_MSGAPI
     // This will ONLY work with a single buffer!
     
-    address -> addressLength = sizeof (address -> address);
+    peerAddress -> peerAddressLength = sizeof (address -> address);
```

Comment:
Need to update `sizeof (address -> address)` too, right?

Reply:
Yeah sorry. Was juggling multiple submodules and my commit got silly ill update it